openvinotoolkit/openvino
# Azure CI: enable MO unit tests for openvino-lin

### Details:
 - Steps are inherited from 'Python API Tests' step from same file
 - Some MO unit tests require availability of 'mock frontend' thus need build of OpenVINO with '-DENABLE_TESTS=ON'
 - Added install rules for 'Mock MO frontend' and its Python bindings to be reachable by unit tests
 
How to verify that it works:
- Check "openvino-lin" CI job. Verify that tests related to 'frontend_ngraph_test.py' are executed and not skipped

Additional time required to install MO packages and execute MO Unit tests is less than 1 minute

### Tickets:
 - 59499
